roofit/roofitcore/src/TestStatistics/RooUnbinnedL.cxx

```diff
@@ -83,6 +90,9 @@ RooUnbinnedL::evaluatePartition(Section events, std::size_t /*components_begin*/
    ROOT::Math::KahanSum<double> result;
    double sumWeight;
 
+   // Do not reevaluate likelihood if parameters have not changed
+   if (!_paramTracker->hasChanged(true) & (_cachedResult != 0)) return _cachedResult;
```

Comment:
```suggestion
   if (!pdf_->isValueDirty()) return _cachedResult;
```
Is it really necessary to introduce overhead with this change tracker? Why can't we just check if the value of the pdf is dirty and needs to be recomputed, in which case the likelihood also needs to be recomputed?

Reply:
Resolved, as discussed below.